tests/by-util/test_tail.rs

```diff
@@ -266,8 +241,12 @@ fn test_follow_redirect_stdin_name_retry() {
 }
 
 #[test]
-#[cfg(not(target_os = "macos"))] // See test_stdin_redirect_dir_when_target_os_is_macos
-#[cfg(all(unix, not(any(target_os = "android", target_os = "freebsd"))))] // FIXME: fix this test for Android/FreeBSD
+#[cfg(all(
+    not(target_vendor = "apple"),
+    not(target_os = "windows"),
+    not(target_os = "android"),
+    not(target_os = "freebsd")
+))] // FIXME:
```

Comment:
There's a couple of these `FIXME:` comments. I'd say either give an explanation or remove the colon :)

Reply:
However, I'll write a short explanation for the standalone FIXMEs :)